TwitterPolling/twitter-authorize.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+#-----------------------------------------------------------------------
+# twitter-authorize:
+#  - step through the process of creating and authorization a
+#    Twitter application.
+#-----------------------------------------------------------------------
+
+import twitter
+import time
+
+print("1. Create a new Twitter application at: https://apps.twitter.com\n")
+
+print("2. When you have created the application, enter:")
+print("   your application name:", end=' ')
+app_name = input()
+
+print("   your consumer key:", end=' ')
+consumer_key = input()
+
+print("   your consumer secret:", end=' ')
+consumer_secret = input()
+
+print()
+print("3. Now, we need to authorize this application.")
+print("   You'll be forwarded to a web browser in two seconds.\n")
+
+time.sleep(2)
+
+access_key, access_secret = twitter.oauth_dance(app_name, consumer_key, consumer_secret)
+
+print()
+print("4. Now, replace the credentials in config.py with the below:\n")
```

Comment:
Ah. This is where you can just use a utility and write it to the file for them. I'd recommend 
```
with open('config', 'w') as file:
  file.write(""consumer_key = '%s'" % consumer_key)...
```

Reply:
Let's improve this auth file in the next PR xD

Reply:
haha ok